resources/config.rb

```diff
@@ -24,7 +24,7 @@
 property :debug, [true, false, 0, 1], default: false, coerce: proc { |v| v.to_i }
 property :dont_blame_nrpe, [true, false, 0, 1], default: false, coerce: proc { |v| v.to_i }
 property :include_dir, String, default: '/etc/nagios/nrpe.d'
-property :pid_file, String, default: '/var/run/nrpe.pid'
+property :pid_file, String, default: '/var/run/nrpe/nrpe.pid'
```

Comment:
What was the problem on CentOS 7 platform?

Reply:
The travis-ci run failed on CentOS 7 with the following message

     ✔  type should eq "systemd"
  Processes nrpe
     ∅  entries.length should eq 1
     
     expected: 1
          got: 0
     
     (compared using ==)
     
     ∅  users should eq ["nrpe"]
     
     expected: ["nrpe"]
          got: []
     
     (compared using ==)
     
  Group nrpe


----

This is the summary:
Test Summary: 18 successful, 3 failures, 0 skipped
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Verify failed on instance <default-centos-7>.  Please see .kitchen/logs/default-centos-7.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
The command "kitchen verify ${INSTANCE}" exited with 20.
after_script
0.01s$ cat .kitchen/logs/kitchen.log
I, [2017-09-26T19:11:46.333384 #4919]  INFO -- Kitchen: -----> Starting Kitchen (v1.17.0)
I, [2017-09-26T19:11:47.480687 #4919]  INFO -- Kitchen: -----> Creating <default-centos-7>...
I, [2017-09-26T19:12:13.483845 #4919]  INFO -- Kitchen: -----> Converging <default-centos-7>...
I, [2017-09-26T19:12:51.714019 #4919]  INFO -- Kitchen: -----> Setting up <default-centos-7>...
I, [2017-09-26T19:12:51.725492 #4919]  INFO -- Kitchen: -----> Verifying <default-centos-7>...
E, [2017-09-26T19:12:53.427984 #4919] ERROR -- Kitchen: ------Exception-------
E, [2017-09-26T19:12:53.428026 #4919] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2017-09-26T19:12:53.428044 #4919] ERROR -- Kitchen: Message: 1 actions failed.
>>>>>>     Verify failed on instance <default-centos-7>.  Please see .kitchen/logs/default-centos-7.log for more details
E, [2017-09-26T19:12:53.428056 #4919] ERROR -- Kitchen: ----------------------
E, [2017-09-26T19:12:53.428066 #4919] ERROR -- Kitchen: ------Backtrace-------
E, [2017-09-26T19:12:53.428075 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:183:in `report_errors'
E, [2017-09-26T19:12:53.428086 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:174:in `run_action'
E, [2017-09-26T19:12:53.428096 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/action.rb:36:in `block in call'
E, [2017-09-26T19:12:53.428113 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2017-09-26T19:12:53.428137 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command/action.rb:34:in `call'
E, [2017-09-26T19:12:53.428203 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:53:in `perform'
E, [2017-09-26T19:12:53.428238 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:187:in `block (2 levels) in <class:CLI>'
E, [2017-09-26T19:12:53.428293 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
E, [2017-09-26T19:12:53.428342 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
E, [2017-09-26T19:12:53.428392 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/cli.rb:334:in `invoke_task'
E, [2017-09-26T19:12:53.428412 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
E, [2017-09-26T19:12:53.428432 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
E, [2017-09-26T19:12:53.428446 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/bin/kitchen:13:in `block in <top (required)>'
E, [2017-09-26T19:12:53.428474 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/errors.rb:171:in `with_friendly_errors'
E, [2017-09-26T19:12:53.428505 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/bin/kitchen:13:in `<top (required)>'
E, [2017-09-26T19:12:53.428519 #4919] ERROR -- Kitchen: /opt/chefdk/bin/kitchen:267:in `load'
E, [2017-09-26T19:12:53.428528 #4919] ERROR -- Kitchen: /opt/chefdk/bin/kitchen:267:in `<main>'
E, [2017-09-26T19:12:53.428537 #4919] ERROR -- Kitchen: ----End Backtrace-----
E, [2017-09-26T19:12:53.428546 #4919] ERROR -- Kitchen: -Composite Exception--
E, [2017-09-26T19:12:53.428555 #4919] ERROR -- Kitchen: Class: Kitchen::InstanceFailure
E, [2017-09-26T19:12:53.428564 #4919] ERROR -- Kitchen: Message: Verify failed on instance <default-centos-7>.  Please see .kitchen/logs/default-centos-7.log for more details
E, [2017-09-26T19:12:53.428574 #4919] ERROR -- Kitchen: ----------------------
E, [2017-09-26T19:12:53.428583 #4919] ERROR -- Kitchen: ------Backtrace-------
E, [2017-09-26T19:12:53.428598 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-inspec-0.19.0/lib/kitchen/verifier/inspec.rb:96:in `call'
E, [2017-09-26T19:12:53.428610 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:434:in `block in verify_action'
E, [2017-09-26T19:12:53.428630 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:527:in `synchronize_or_call'
E, [2017-09-26T19:12:53.428642 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:489:in `block in action'
E, [2017-09-26T19:12:53.428656 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2017-09-26T19:12:53.428679 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:488:in `action'
E, [2017-09-26T19:12:53.428752 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:426:in `verify_action'
E, [2017-09-26T19:12:53.428766 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:359:in `block in transition_to'
E, [2017-09-26T19:12:53.428780 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:358:in `each'
E, [2017-09-26T19:12:53.428792 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:358:in `transition_to'
E, [2017-09-26T19:12:53.428801 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:157:in `verify'
E, [2017-09-26T19:12:53.428812 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:197:in `public_send'
E, [2017-09-26T19:12:53.428824 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2017-09-26T19:12:53.428856 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2017-09-26T19:12:53.428866 #4919] ERROR -- Kitchen: ----End Backtrace-----
E, [2017-09-26T19:12:53.428874 #4919] ERROR -- Kitchen: ---Nested Exception---
E, [2017-09-26T19:12:53.428883 #4919] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2017-09-26T19:12:53.428892 #4919] ERROR -- Kitchen: Message: Inspec Runner returns 1
E, [2017-09-26T19:12:53.428900 #4919] ERROR -- Kitchen: ----------------------
E, [2017-09-26T19:12:53.428908 #4919] ERROR -- Kitchen: ------Backtrace-------
E, [2017-09-26T19:12:53.428917 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/kitchen-inspec-0.19.0/lib/kitchen/verifier/inspec.rb:96:in `call'
E, [2017-09-26T19:12:53.428992 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:434:in `block in verify_action'
E, [2017-09-26T19:12:53.429031 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:527:in `synchronize_or_call'
E, [2017-09-26T19:12:53.429049 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:489:in `block in action'
E, [2017-09-26T19:12:53.429062 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
E, [2017-09-26T19:12:53.429073 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:488:in `action'
E, [2017-09-26T19:12:53.429158 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:426:in `verify_action'
E, [2017-09-26T19:12:53.429175 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:359:in `block in transition_to'
E, [2017-09-26T19:12:53.429189 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:358:in `each'
E, [2017-09-26T19:12:53.429200 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:358:in `transition_to'
E, [2017-09-26T19:12:53.429209 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/instance.rb:157:in `verify'
E, [2017-09-26T19:12:53.429218 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:197:in `public_send'
E, [2017-09-26T19:12:53.429228 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2017-09-26T19:12:53.429237 #4919] ERROR -- Kitchen: /opt/chefdk/embedded/lib/ruby/gems/2.4.0/gems/test-kitchen-1.17.0/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2017-09-26T19:12:53.429246 #4919] ERROR -- Kitchen: ----End Backtrace-----
Done. Your build exited with 1.